projects/admin-api/src/lib/models/authorization-rules.model.ts

```diff
@@ -0,0 +1,18 @@
+export enum AuthorizationRuleDecision {
+    ALLOW = 'allow',
+    DENY = 'deny',
+}
+
+export interface AuthorizationRule {
```

Comment:
I think we could lose this extra interface. Why not use just

```
export interface AuthorizationRuleGroup {
  groupName: string;
  decisions: { [name: string]: AuthorizationRuleDecision },
}
```

Reply:
Mostly for future extensibility and not having to run a migration, since this interface is shared between api and admin.